core/src/main/java/nl/nn/adapterframework/jta/StatusRecordingTransactionManager.java

```diff
@@ -63,6 +63,7 @@
 	 * @return true if any transactions are pending, that need to be recovered later. 
 	 */
 	protected abstract TransactionManager createTransactionManager() throws TransactionSystemException;
+	//TODO make it so the boolean represents whether the action 'shutdown' was successful.
```

Comment:
Het gaat niet om 'successful' of niet, maar of er nog transacties 'in flight' zijn. Dat wil zeggen: transacties waarvan besloten is dat ze gecommit gaan worden (first phase), maar waarvoor nog niet alle resource managers hebben aangegeven dat de transactie voltooid is.

Reply:
Voor de methode shutdown maakt dat toch niet uit. Die doet of een shutdown, of niet.